tests/integration/src/lib.rs

```diff
@@ -110,8 +110,6 @@ mod tests {
         let token = MockERC20::new(token_address, &signer);
         let initial_balance = token.balanceOf(claimer_address).call().await.unwrap()._0;
 
-        assert!(initial_balance == U256::ZERO);
```

Comment:
Why remove this check?

Reply:
because now the initial  balance of operator 2 is not zero. We are funding it in this line
`FundOperator.fundOperator(address(erc20Mock), avsconfig.operator2Addr, 10e18);`